forge-game/src/main/java/forge/game/keyword/Keyword.java

```diff
@@ -112,7 +112,7 @@ public enum Keyword {
     INTIMIDATE("Intimidate", SimpleKeyword.class, true, "This creature can't be blocked except by artifact creatures and/or creatures that share a color with it."),
     KICKER("Kicker", Kicker.class, false, "You may pay an additional %s as you cast this spell."),
     JUMP_START("Jump-start", SimpleKeyword.class, false, "You may cast this card from your graveyard by discarding a card in addition to paying its other costs. Then exile this card."),
-    LANDWALK("Landwalk", KeywordWithType.class, false, "This creature is unblockable as long as defending player controls a %s."),
+    LANDWALK("Landwalk", KeywordWithType.class, false, "This creature is unblockable as long as defending player controls {1:%s}."),
```

Comment:
Can `KeywordInstance.redundant()` logic be improved to check for type too now?

might be good for GUI + engine checks

> 702.14e. Multiple instances of the same kind of landwalk on the same creature are redundant.

Reply:
we probably might need to find a better logic for "redundant" checks

Reply:
i refactored the Redundant Check